### PR TITLE
FW-2294 Show categories in FAQ within chat widget

### PR DESCRIPTION
**Pull request by Prochnost55**
### What do you want to achieve?
- Add a section in FAQ where the user will see categories as shown in helpcenter

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- Go to faq
- various categories should appear

https://user-images.githubusercontent.com/19753380/129435495-3e8f672d-de60-49e3-b93d-677f8a7cb78e.mov

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

**Comment by devashishmamgain** [comment]
@Prochnost55 is it deployed anywhere for testing? Or can you attach screenshot of it

**Comment by Prochnost55** [comment]
> @Prochnost55 is it deployed anywhere for testing? Or can you attach screenshot of it

Add a screen recording

File: webplugin/js/app/kommunicate-ui.js [hunk 1]
```diff
@@ -491,6 +415,33 @@ KommunicateUI = {
             );
             MCK_BOT_MESSAGE_QUEUE = [];
             if (MCK_EVENT_HISTORY.length >= 2) {
+                if(
+                    MCK_EVENT_HISTORY[MCK_EVENT_HISTORY.length - 2] ==
+                    'km-faq-category-list'
+                ){
+                    KommunicateUI.showHeader();
+                    $applozic('#km-faqdiv')
```
Comment (devashishmamgain) [inline webplugin/js/app/kommunicate-ui.js:423]: what is common between all these elements may be parent container, or something else? if not, then can we make something common such as adding a specific class
then replace this code with one just one of code that add and remove class based on that common identifer.

File: webplugin/js/app/kommunicate-ui.js [hunk 2]
```diff
@@ -601,8 +552,126 @@ KommunicateUI = {
...
+                                kommunicateCommons.formatHtmlTag(faqDetails.title);
+                            // FAQ description is already coming in formatted way from the dashboard FAQ editor.
+                            $applozic('#km-faqanswer').append(
+                                '<div class="km-faqanswer-list ql-snow"><div class="km-faqquestion">' +
+                                    faqTitle +
+                                    '</div> <div class="km-faqanchor km-faqanswer ql-editor">' +
+                                    faqDetails.body +
+                                    '</div></div>'
+                            );
+                            $applozic('#km-contact-search-input-box')
```
Comment (devashishmamgain) [inline webplugin/js/app/kommunicate-ui.js:601]: same comment as above

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

**Comment by Prochnost55** [comment]
Used modifyClassList function to make code cleaner.
Also, added faq-common and km-faqdiv to toggle FAQ

**Review by devashishmamgain — APPROVED** [review]

**Review by ChannuBS — DISMISSED** [review]

**Review by ChannuBS — APPROVED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Pull request merged**